setup.py

```diff
@@ -136,6 +136,9 @@ def run_tests(self):
         'invenio_base.apps': [
             'invenio_pidstore = invenio_pidstore:InvenioPIDStore',
         ],
+        'invenio_admin.register': [
```

Comment:
Hmmm..... not sure - I agree ``invenio_admin.register`` is not the best.

Alternatives:
```
invenio_admin.init_admins
invenio_admin.init
invenio_admin.admins
invenio_admin.exts
``` 

The entry point specifies ``callable``s that initialise the admin.